test/e2e/upgrades/apparmor.go

```diff
@@ -52,6 +55,7 @@ func (t *AppArmorUpgradeTest) Setup(f *framework.Framework) {
 // Test waits for the upgrade to complete, and then verifies that a
 // pod can still consume the secret.
 func (t *AppArmorUpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade UpgradeType) {
+	common.SkipIfAppArmorNotSupported()
```

Comment:
I think this will still mark the whole upgrade test skipped since ultimately `Setup()` and `Test()` are called in the same wrapper function:

https://github.com/kubernetes/kubernetes/blob/8fd414537b5143ab039cb910590237cabf4af783/test/e2e/chaosmonkey/chaosmonkey.go#L67